src/v0/util/regulation-api.js

```diff
@@ -8,10 +8,7 @@ class RegulationApiUtils {
    */
   static executeCommonValidations(userAttributes) {
     if (!Array.isArray(userAttributes)) {
-      throw new ErrorBuilder()
-        .setMessage("userAttributes is not an array")
-        .setStatus(400)
-        .build();
+      throw new PlatformError("userAttributes is not an array");
```

Comment:
```suggestion
      throw new PlatformError("[Regulation API] userAttributes is not an array");
```

Reply:
@utsabc we are not adding this to message as context will be added by server but since is this not a destination, what metadata does server add for this one?

Reply:
User deletion is for destinations. Hence, the destination type stat tag is added at the versioned router level in the transformer.

Reply:
Also, I just realized that it's actually an `InstrumentationError` rather than `PlatformError`. I'll fix this.

Reply:
Why `InstrumentationError` this request is coming from server

Reply:
This is platform only

Reply:
@utsabc, the context of user deletion is already captured at the stat tags level, `feature: "userDeletion"`. So, this won't be required.

Reply:
However, sending stat tags isn't part of the data contract for the regulation API. So, once we add it, this will be addressed.

Reply:
> Why `InstrumentationError` this request is coming from server

@utsabc, users would send this data, right?

Reply:
No server is forming this request: https://github.com/rudderlabs/rudder-server/blob/37f9bb37a943a37d4d7fee6e69fbcf164ba5c181/regulation-worker/internal/delete/api/api.go#L170

Reply:
Addressed.